charts/api-syncagent/templates/deployment.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: '{{ template "name" . }}'
+  labels:
+    app.kubernetes.io/name: kcp-sync-agent
+    app.kubernetes.io/instance: '{{ template "agentname" . }}'
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ template "name" . }}'
+spec:
+  replicas: {{ .Values.replicas | default 1 }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kcp-sync-agent
```

Let's keep this consistent, should be

```suggestion
      app.kubernetes.io/name: kcp-api-syncagent
```